locations.txt

```diff
@@ -334,3 +335,4 @@ Ouagadougou, Burkina Faso
 Motihari, Bihar, India
 Ottawa, Ontario, Canada
 Bangalore, India
+
```

Comment:
This is a mistake, I should have removed the line

Reply:
Yeah you should've....

Reply:
Go away @jacobbamberger  <3